app/models/worldwide_organisation_edition.rb

```diff
@@ -0,0 +1,7 @@
+class WorldwideOrganisationEdition < Edition
+  include Edition::Organisations
```

Comment:
This is interesting, I think all editions have related organisations which can be filtered on here 👇 We think this is desirable and we would probably migrate the existing ones all to be FCDO
![image](https://github.com/alphagov/whitehall/assets/47089130/e0e983fe-7a00-4e5b-a2d5-e1c7012549a1)

Reply:
There are a few of these that are owned by other organisations, I think the MOD have a couple so we'd have to ask around a bit before making this decision

Reply:
Ah, so these might replace the "sponsoring organisations"?